cmd/preview/deploy.go

```diff
@@ -137,7 +137,7 @@ func (pw *Command) deployPreview(ctx context.Context, opts *DeployOptions) (*typ
 		})
 	}
 
-	return pw.okClient.Previews().DeployPreview(ctx, opts.name, opts.scope, opts.repository, opts.branch, opts.sourceUrl, opts.filename, varList)
+	return pw.okClient.Previews().DeployPreview(ctx, opts.name, opts.scope, opts.repository, opts.branch, opts.sourceUrl, opts.file, varList)
```

Comment:
please also add a unit test to ensure we catch any regressions

Reply:
There is no easy way to test it... There are two different functions that already have all tests done correctly. The problem here was more that we were not using the correct parameter. We could create a test that tests both functions one after the other but we won't get if the cfmap is getting the correct filename since is something done in the backend side

Reply:
There's probably an integration test that we can look at but that can be follow-up as well.